plugins/catalog-react/src/hooks/useRelatedEntities.ts

```diff
@@ -15,36 +15,48 @@
  */
 import { Entity } from '@backstage/catalog-model';
 import { useApi } from '@backstage/core';
-import { catalogApiRef } from '@backstage/plugin-catalog-react';
 import { useAsyncRetry } from 'react-use';
+import { catalogApiRef } from '../api';
 
-// TODO: Maybe this hook is interesting for others too?
 export function useRelatedEntities(
   entity: Entity,
-  type: string,
+  { type, kind }: { type?: string; kind?: string },
 ): {
-  entities: (Entity | undefined)[] | undefined;
+  entities: Entity[] | undefined;
   loading: boolean;
   error: Error | undefined;
 } {
   const catalogApi = useApi(catalogApiRef);
-  const { loading, value, error } = useAsyncRetry<
-    (Entity | undefined)[]
+  const { loading, value: entities, error } = useAsyncRetry<
```

Comment:
No need to use the `Retry` variation if you don't use that facility right? 🤔

Reply:
Yep, we don't need that here.